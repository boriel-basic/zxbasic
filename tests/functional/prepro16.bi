' Must return:
' PRINT 20;
' PRINT 10 + 20;


#define macro(x) PRINT x;
#define y 20

macro(y)
macro(10 + y)


