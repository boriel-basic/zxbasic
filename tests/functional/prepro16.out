#line 1 "prepro16.bi"








PRINT 20;
PRINT 10 + 20;


